docs/src/pages/components/badge.mdx

```diff
@@ -1,2 +1,152 @@
 # Badge
 
+The Badge component displays a dot or a circle with a count. It is normally used to indicate pending items (items in cart, unread messages, etc.)
+
```

Comment:
I think it'd be a good idea to explain that the `<Cart />` component that you're using is from Anchor and provide a link to that section. Assume the person has no idea where any of these components are coming from.